### PR TITLE
ci: Update linting dependencies, and setup eslint-plugin-unicorn (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
This should fail until #6041 is merged

### CODE REVIEW

**Review by ivov — APPROVED** [review]
LGTM, pending #6041. (PR title has a typo.)

Will also update dependencies in the nodelinter once we merge this, to keep them in sync.

**Review by ivov — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.226.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.226.1)

**Pull request merged**
